client/src/components/DataEntry/index.test.tsx

```diff
@@ -265,5 +257,114 @@ describe('DataEntry', () => {
         )
       })
     })
+
+    it('audits ballots', async () => {
+      const checkMockApi = mockApi([
+        apiCalls.auditBoardMe,
+        apiCalls.contests,
+        apiCalls.ballotsInitial,
+        apiCalls.auditBallot('ballot-id-2', {
+          status: 'AUDITED',
+          interpretations: [
+            {
+              contestId: 'contest-id-1',
+              interpretation: 'VOTE',
+              choiceIds: ['choice-id-1'],
+              comment: null,
+            },
+            {
+              contestId: 'contest-id-2',
+              interpretation: 'CANT_AGREE',
+              choiceIds: [],
+              comment: null,
+            },
+          ],
+        }),
+        apiCalls.ballotsOneAudited,
+      ])
```

Comment:
Here's how that new mock helper gets used

Reply:
Interesting! It looks more neat, but it's also not immediately apparent what's going on. I'll have to dig into it and play with it myself before I grok it completely.